dix/req.c

```diff
@@ -2334,9 +2334,12 @@ M0_INTERNAL int m0_dix_put(struct m0_dix_req      *req,
 
 	M0_PRE(keys->ov_vec.v_nr == vals->ov_vec.v_nr);
 	M0_PRE(keys_nr != 0);
-	/* Only overwrite, crow, sync_wait and skip_layout flags are allowed. */
+	/*
+	 * Only overwrite, crow, sync_wait, skip_layout, no_dtm
```

Comment:
What's the point of repeating the flag names in the comment?
Suggestion: "Only the mentioned below flags are allowed."

Reply:
I agree.

Reply:
fixed.